internal/handlers/pg/msg_getlog.go

```diff
@@ -83,14 +83,27 @@ func (h *Handler) MsgGetLog(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		))
 
 	case "startupWarnings":
+		// TODO telementry
 		pv, _, _ := strings.Cut(h.StateProvider.Get().HandlerVersion, " ")
 		mv := version.Get()
 
+		startupWarnings := []string{
+			"Powered by FerretDB " + mv.Version + " and PostgreSQL " + pv + ".",
+			"Please star us on GitHub: https://github.com/FerretDB/FerretDB.",
+		}
+
+		state := h.StateProvider.Get()
+		if state.Telemetry == nil {
+			startupWarnings = append(
+				startupWarnings,
+				"The telemetry state is undecided; the first report will be sent soon. "+
```

Comment:
Just want to double check, not showing how long we wait until the first report is sent isn't an issue here and tigris.

Reply:
To implement this, we need to know the delay. My first implementation contained the delay being present in the `State`, but @AlekSi said it's better not to have delay time in the warning than to have it stored in the `State`. So, this is why :)

Reply:
I'm not 100% sure what state _is_, but I'm sure we don't want to be a grab-bag of everything, including CLI flags.